src/ja/2017-03/info.yml

```diff
@@ -0,0 +1,6 @@
+---
+  title: "ガラテヤの信徒への手紙における副音"
+  description: "すいませんけど、記述が当分の間オンラインにありません。"
+  human_date: "2017年3期の聖書研究ガイド"
+  start_date: "24/06/2017"
+  end_date: "22/09/2017"
```

Comment:
@imasaru Check out `color_primary` and `color_primary_dark` in the `info.yml` for `en`